docs/content/tutorials/ingress-proxy-set-headers-annotation.md

```diff
@@ -0,0 +1,364 @@
+---
```

Comment:
I don't think we need a tutorial for this annotation. The inline annotation documentation in the Advanced Configuration with Annotations page should really be enough to communicate what it does IMO.